homeassistant/components/sensor/rtorrent.py

```diff
@@ -28,7 +33,7 @@
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
     vol.Required(CONF_URL): cv.url,
     vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
-    vol.Optional(CONF_MONITORED_VARIABLES, default=[]): vol.All(
+    vol.Optional(CONF_MONITORED_VARIABLES, default=ALL_SENSOR_TYPES): vol.All(
```

Comment:
default can just be `SENSOR_TYPES`, as iterating over a dict creates list of keys.